controllers/constants/constants.go

```diff
@@ -69,10 +72,18 @@ const (
 	CSINodeSCCClusterRole                 = "node-scc-clusterrole"
 	CSINodeSCCClusterRoleBinding          = "node-scc-clusterrolebinding"
 
-	RcloneRetainStorageClass = "ibm-object-storage-rclone-retain-sc"
-	RcloneStorageClass       = "ibm-object-storage-rclone-sc"
-	S3fsRetainStorageClass   = "ibm-object-storage-s3fs-retain-sc"
-	S3fsStorageClass         = "ibm-object-storage-s3fs-sc"
+	StorageClassPrefix = "ibm-object-storage-"
+	StorageClassSuffix = "-sc"
```

Comment:
I think suffix `-sc` will be removed.
Check with @nkkashyap

Reply:
This will be taken up with next PR for adding SCs.